core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.spec.ts

```diff
@@ -263,6 +263,20 @@ describe('Topics and skills dashboard page component', () => {
     expect(componentInstance.filterBoxIsShown).toBeTrue();
   });
 
+  it('should display filter box on maximizing the window', () => {
+    componentInstance.filterBoxIsShown = false;
+    spyOn(windowDimensionsService, 'isWindowNarrow').and.returnValue(false);
+    componentInstance.filterBoxOnResize();
+    expect(componentInstance.filterBoxIsShown).toBeTrue();
```

Comment:
```suggestion
    componentInstance.filterBoxIsShown = false;
 
    spyOn(windowDimensionsService, 'isWindowNarrow').and.returnValue(false);
 
    componentInstance.filterBoxOnResize();

    expect(componentInstance.filterBoxIsShown).toBeTrue();
```

Reply:
@gp201 I've commited the requested changes. PTAL. Thanks!